### ISSUE TITLE
The Effects of Shuffle Tiles in the Fine-tuning of Slide Encoder

### ISSUE THREAD

**Issue by shuangyanl** [prov-gigapath/prov-gigapath#91]
Hello, the fine-tuning code seems to use the same config for `shuffle_tiles` in args.task_config when loading train, validation and test data using SlideDataset ([here](https://github.com/prov-gigapath/prov-gigapath/blob/98b8c6674aaad1f3a41ea57b7f59d9b65c9f233c/finetune/main.py#L74)). If I set `shuffle_tiles` to true, it applies to validation and test data loading as well as train data loading. 

I understand it can add randomness to data when training, but I'm not sure why shuffling tiles are needed for validation and testing given that we may test on all tiles of a slide and the order of tiles seems not relevant in these situations.

What does the shuffling tiles actually do? How does it affect the training, validation and testing processes? I think I may have missed something here. Hope you can help clarify. Thanks!

Relevant code:
slide_dataset.py L182 ([here](https://github.com/prov-gigapath/prov-gigapath/blob/98b8c6674aaad1f3a41ea57b7f59d9b65c9f233c/finetune/datasets/slide_datatset.py#L181)):
